paper.tex

```diff
@@ -832,19 +839,16 @@ \subsection{Population-level features, not genome features}
 demographic structure is the common thread connecting many widely used
 population genetic methods, many features of underlying genome biology are
 omitted. Most notably, mutation and recombination rates are left unspecified
-(though they can be encoded as metadata in the data structure). Genome ploidy
-is left ambiguous, and selection and dominance models are absent. Using a Demes
-model, which specifies deterministic population sizes, as the demographic input
-for simulation using software that allows for selection implies that selection
-is \emph{soft}, that is, genome fitnesses are rescaled relative to the mean
-population fitness \citep{christiansen1975hard}. It is possible to instead
-interpret population sizes as carrying capacities, allowing for a model of
-\emph{hard} selection. However, this should not be understood as the default
-interpretation of population sizes, which conventionally specifies a
-deterministic number of individuals. Finally, as Demes assumes exchangeability
-of individuals within populations, it doesn't provide a way to define geographic
-spatial structure within populations.
-
+(though they can be encoded as metadata in the data structure), as are
+sequence length or annotations or genomic features.
+Genome ploidy is left ambiguous, and selection and dominance models are absent. 
```

Comment:
"Ambiguous" doesn't feel like the right word here. We're counting individuals, and their ploidy is external to the model, like other genomic features. Maybe just add to the previous sentence, 

> as are properties of genomes such as sequence length, gene annotations and ploidy.